src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -389,6 +389,7 @@ private void updateAccountDetails() {
     if (rVal == CreateUpdateAccountPanel.ReturnValue.CANCEL) {
       return;
     }
+    //TODO: Replace MD5Crypt.crypt with a decryption algorithm
```

Comment:
Why 'decryption' algorithm? I'm not sure where we would be decrypting, which makes me  wonder.